src/components/SplitView/index.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
```

Comment:
It would make sense to split this into a `ControlledSplitView` and a light stateful wrapper `SplitView`. There are use cases where fully controlling the open state makes sense.